addons/website_event/controllers/main.py

```diff
@@ -166,8 +166,8 @@ def event_page(self, event, page, **post):
         except ValueError:
             # page not found
             values['path'] = re.sub(r"^website_event\.", '', page)
-            values['from_template'] = 'website_event.default_page'  # .strip('website_event.')
-            page = request.env.user.has_group('website.group_website_designer') and 'website.page_404' or 'http_routing.404'
+            values['from_template'] = 'website_event.default_page&event_id=%s' % event.id
```

Comment:
I did tried that but it was giving me `Method Not Allowed` error, as I believe `/website/add/` route doesn't have `event_id` in param? So I thought its better to add in template instead of modifying that route